packages/atomic/src/components/common/atomic-focus-trap.tsx

```diff
@@ -28,24 +29,6 @@ function contains(
   return parent === null ? false : contains(ancestor, parent);
 }
 
-function getFirstFocusableDescendant(element: Element) {
```

Comment:
I am no longer assuming that the first element in the modal is the close button, since it's optional. Instead, I extracted this function and improved it.